### PR TITLE
feat: add `@nuxt/test-utils` ecosystem test

### PR DESCRIPTION
**Pull request by danielroe**
This adds a test suite for `@nuxt/test-utils`. It should be ✅ out of the box, and hopefully this will mean we can get a heads-up if we need to update or adjust our implementation down the line.

Further review and thoughts also welcome of our implementation. 🙏

### CODE REVIEW

File: .github/workflows/ecosystem-ci.yml [hunk 1]
```diff
@@ -41,6 +41,7 @@ jobs:
       matrix:
         suite:
           - nuxt
+          - nuxt-text-utils
```
Comment (AriPerkkio) [inline .github/workflows/ecosystem-ci.yml:44]:
```suggestion
          - nuxt-test-utils
```

File: .github/workflows/ecosystem-ci-selected.yml [hunk 2]
```diff
@@ -35,6 +35,7 @@ on:
         type: choice
         options:
           - nuxt
+          - nuxt-text-utils
```
Comment (AriPerkkio) [inline .github/workflows/ecosystem-ci-selected.yml:38]:
```suggestion
          - nuxt-test-utils
```

File: .github/workflows/ecosystem-ci-from-pr.yml [hunk 3]
```diff
@@ -96,6 +97,7 @@ jobs:
       matrix:
         suite:
           - nuxt
+          - nuxt-text-utils
```
Comment (AriPerkkio) [inline .github/workflows/ecosystem-ci-from-pr.yml:100]:
```suggestion
          - nuxt-test-utils
```

File: .github/workflows/ecosystem-ci-from-pr.yml [hunk 4]
```diff
@@ -30,6 +30,7 @@ on:
         options:
           - "-"
           - nuxt
+          - nuxt-text-utils
```
Comment (AriPerkkio) [inline .github/workflows/ecosystem-ci-from-pr.yml:33]:
```suggestion
          - nuxt-test-utils
```

**Comment by danielroe** [comment]
Thank you - and thank you for spotting the typo ❤️

**Pull request merged**